v2x_ros_driver/include/v2x_ros_driver/v2x_radio_client.h

```diff
@@ -131,16 +131,17 @@ class V2XRadioClient
      * @brief Checks actual message vector size against size specified in MessageFrame
      * @param msg_vec Full message vector under test
      * @param start_index Starting index for detected payload within message vector
-     * @param end_index Ending index for payload within message vector
      * @param entry Recevied data reference used to generate test vectors
      */
-    bool isValidMsgSize(const std::vector<uint8_t> msg_vec, size_t start_index, size_t end_index, const std::vector<uint8_t> entry);
+    bool isValidMsgSize(const std::vector<uint8_t> &msg_vec, size_t start_index, const std::vector<uint8_t> &entry);
     
     /**
-     * @brief Checks msg_id against Provider Service ID (PSID) list to make sure a Wireless Access in Vehicular Environments (WAVE) Service Advertisement (WSA) with valid PSID is not accidentally used instead of MessageFrame
+     * @brief Checks msg_id against Provider Service ID (PSID) list to make sure a Wireless Access in Vehicular Environments (WAVE) 
+     * Short Message Protocol (WSMP) T-Header with valid PSID is not accidentally used instead of WSM Data (MessageFrame + Payload).
+     * IEEE 1609.3 (2020) - Section 8.3 Figure 33 and Section 8.3.3 Table 21, Figure 36 (TPID = 0). 
      * @param msg_id Message ID under test
      */
-    bool isValidPSID(const std::string& msg_id);
+    bool isValidPSID(const std::string &msg_id);
```

Comment:
Consider changing the input parameter from `msg_id` to `dsrc_msg_id` (also would require an update to the parameter description comment above)

Reply:
I didn't update this msg_id because it could be confusing when it gets compared with the dsrc_msg_id list 
if (msg_id == dsrc_msg_id)